internal/peermanager/peerwantmanager.go

```diff
@@ -19,10 +19,17 @@ type Gauge interface {
 // peerWantManager keeps track of which want-haves and want-blocks have been
 // sent to each peer, so that the PeerManager doesn't send duplicates.
 type peerWantManager struct {
+	// peerWants maps peers to outstanding wants.
+	// A peer's wants is the _union_ of the broadcast wants and the wants in
+	// this list.
 	peerWants map[peer.ID]*peerWant
-	// Reverse index mapping wants to the peers that sent them. This is used
-	// to speed up cancels
+
+	// Reverse index of all wants in peerWants.
 	wantPeers map[cid.Cid]map[peer.ID]struct{}
```

Comment:
Note: this _doesn't_ include all wanted blocks, just those mentioned in peerWants.